tests/test_utils.py

```diff
@@ -1,20 +1,34 @@
+import filecmp
 import os
+import shutil
+from pathlib import Path
 
+import isceobj
 import numpy as np
 import pytest
 from osgeo import gdal
 
 from hyp3_isce2.utils import (
```

Comment:
we're basically importing the entire `utils` module here. Can we convert this to `from hyp3_isce2 import utils` and just modify the function calls?

Reply:
will do it.